intranet/static/js/eighth/statistics.js

```diff
@@ -0,0 +1,66 @@
+/* global $ */
+$(document).ready(function() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'anonymous' has a complexity of 5.](https://www.codacy.com/app/tjhsst-ion/ion/file/2622929497/issues/source?bid=3212528&fileBranchId=3299315#l2)